config/300-pipelinerun.yaml

```diff
@@ -28,16 +28,18 @@ spec:
     - prs
   scope: Namespaced
   additionalPrinterColumns:
-    - JSONPath: .status.conditions[?(@.type=="Succeeded")].status
-      description: The status of the pipelinrun
-      name: Status
+    - name: Succeeded
       type: string
-    - JSONPath: .status.startTime
-      name: StartTime
+      JSONPath: ".status.conditions[?(@.type==\"Succeeded\")].status"
+    - name: Reason
+      type: string
+      JSONPath: ".status.conditions[?(@.type==\"Succeeded\")].reason"
```

Comment:
This could get quite lengthy while printed on the console, and i don't think there is a way to ellipse the string

Reply:
@paassdc kinda agreeing with @chmouel here, maybe we should remove the `Reason` field in here *and* in `300-taskrun.yaml` too, what do you think ? :angel:

Reply:
Agreed @chmouel @vincent-pli 
An alternative is to print the `reason` field on `kubectl get -wide` flag?
More [info](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#priority).

Reply:
In my opinion, I like the "reason", but the "reason" should be well designed.
With that metric it's no need to use "describe" or "-oyaml" to get details.
Actually, the lengthy is not import i think :)

Reply:
Thank you for your comments, while I think the `Reason` is still very useful at sometimes. Agreed @vincent-pli  the `Reason`  should be well designed.

Reply:
I don't have a strong opinion here, what do people think we should do to get this merged?